### PR TITLE
Introduce CmApplication

### PR DESCRIPTION
**Pull request by tomaszdurka**

### CODE REVIEW

**Comment by tomaszdurka** [comment]
@vogdb please review

**Comment by fvovan** [comment]
lgmt but I would add some directory existance validation to `CommandRunner`.

**Comment by tomaszdurka** [comment]
CommandRuner is not dependant on and directory. If you talked bout CmApplication that is fine too - failing command is enough.

Change binaryPath -> path, adjusted readme.
@fvovan please give final yes.

File: lib/index.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ var CMApiClient = require('./cm-api-client');
 var Auth = require('./auth');
 var Streams = require('./streams');
 var Logger = require('./logger');
+var CommandRunner = require('./command-runner');
```
Comment (fvovan) [inline lib/index.js]: I guess it was deleted :p

Comment (tomaszdurka) [inline lib/index.js]: Yes, done.

**Comment by fvovan** [comment]
lgtm

**Pull request merged**